components/esp_modem/examples/modem_tcp_client/main/CMakeLists.txt

```diff
@@ -0,0 +1,5 @@
+idf_component_register(SRCS "modem_client.cpp"
+                            "sock_dce.cpp"
+                            "sock_commands.cpp"
+                       INCLUDE_DIRS ".")
+target_compile_options(${COMPONENT_LIB} PRIVATE "-Wno-format")
```

Comment:
Still necessary?

Reply:
Unfortunately it is. Still in my pipeline and plan to work on it :-( 
But closing this discussion, as unrelated to these changes.